satpy/tests/reader_tests/test_mersi_l1b.py

```diff
@@ -338,32 +396,66 @@ def _num_cols_for_file_type(self):
 
     @property
     def _geo_prefix_for_file_type(self):
-        if "1000" in self.filetype_info["file_type"]:
-            return "Geolocation/"
-        elif "500" in self.filetype_info["file_type"]:
-            return "Geolocation/"
-        else:
+        if self.filetype_info["file_type"].startswith(("fy3a_mersi1", "fy3b_mersi1")):
             return ""
+        else:
+            if "1000" in self.filetype_info["file_type"]:
+                return "Geolocation/"
+            elif "500" in self.filetype_info["file_type"]:
+                return "Geolocation/"
+            else:
+                return ""
 
 
-def _test_helper(res):
+def _test_helper(res, band_list, exp_result):
```

Comment:
Great idea, could this function just be renamed to something more descriptive, eg `assert_bands_metadata_as_expected`?